iam_units/test_all.py

```diff
@@ -50,6 +50,11 @@ def test_units(unit_str, dim, new_def):
     assert registry('1 ' + unit_str).dimensionality == dim
 
 
+def test_orders_of_magnitude_14():
+    # The registry recognizes units prefixed by an order of magnitude
+    assert registry('billion EUR').to('million EUR').magnitude == 1e3
```

Comment:
Can the input expression be '1.0 billion EUR'? This makes it clear we're intending to support the case where the user gives a magnitude and considers "billion" as part of the unit expression, rather than as a synonym for "1e9 EUR".

Reply:
added a magnitude